android/app/src/main/java/com/reactnativenavigation/params/parsers/SlidingOverlayParamsParser.java

```diff
@@ -11,6 +11,9 @@ public SlidingOverlayParams parse(Bundle bundle) {
         final SlidingOverlayParams result = new SlidingOverlayParams();
         result.screenInstanceId = bundle.getString("screen");
         result.navigationParams = new NavigationParams(bundle.getBundle("navigationParams"));
+        result.autoDismissTimerSec = bundle.containsKey("autoDismissTimerSec")
+                ? bundle.getInt("autoDismissTimerSec")
+                : null;
```

Comment:
Aren't we always passing `autoDismissTimerSec` from js?
`params.autoDismissTimerSec || 5;`
In that case no need to check if bundle contains key etc and we can avoid null checks.